common/test/acceptance/pages/studio/index.py

```diff
@@ -5,18 +5,31 @@
 from bok_choy.page_object import PageObject
 from . import BASE_URL
 from selenium.webdriver import ActionChains
+from ..common.utils import click_css
 
 
 class DashboardPage(PageObject):
     """
     Studio Home page
     """
 
+    def __init__(self, browser):
+        """
```

Comment:
@attiyaIshaque I still don't understand why we need an init. Won't it automatically call its parent class's init without having to specify it?